src/Umbraco.ModelsBuilder/Umbraco/PureLiveModelFactory.cs

```diff
@@ -182,14 +197,27 @@ private void RazorBuildProvider_CodeGenerationStarted(object sender, EventArgs e
         // tells the factory that it should build a new generation of models
         private void ResetModels()
         {
-            _logger.Logger.Debug<PureLiveModelFactory>("Resetting models.");
+            _logger.Debug<PureLiveModelFactory>("Resetting models.");
 
             try
             {
                 _locker.EnterWriteLock();
 
                 _hasModels = false;
                 _pendingRebuild = true;
+
+                var modelsDirectory = _config.ModelsDirectory;
+                if (!Directory.Exists(modelsDirectory))
+                    Directory.CreateDirectory(modelsDirectory);
+
+                // clear stuff
+                var modelsHashFile = Path.Combine(modelsDirectory, "models.hash");
+                //var modelsSrcFile = Path.Combine(modelsDirectory, "models.generated.cs");
+                //var projFile = Path.Combine(modelsDirectory, "all.generated.cs");
+                var dllPathFile = Path.Combine(modelsDirectory, "all.dll.path");
```

Comment:
This seems odd - not sure if this is intended but since we are supporting PureLive and LiveAppData, there is no "DLL" but there is generated code, need to investigate to see if this is a mistake. Same goes with removing the files from being hashed.

Reply:
This is how the source is in the orig MB too so it's been working like that for a long time. The reason is because we don't need to take into account these commented out files when checking if models need to be rebuilt and they can be left there for reference just in case.